src/main/java/com/github/dockerjava/api/model/Bind.java

```diff
@@ -90,7 +110,7 @@ public int hashCode() {
      */
     @Override
     public String toString() {
-        return path + ":" + volume.getPath() + ":" + accessMode.toString();
+        return path + ":" + volume.getPath() + ":" + accessMode.toString() + (secMode != SELContext.none ? "," + secMode.toString() : "");
```

Comment:
This will not work. The format of this string must be understandable to docker daemon.